author_handlers.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/Financial-Times/go-fthealth/v1a"
+	log "github.com/Sirupsen/logrus"
+	"github.com/gorilla/mux"
+	"net/http"
+)
+
+type authorHandler struct {
+	authorsService authorsService
+	transformer    transformer
+}
+
+func newAuthorHandler(as authorsService) authorHandler {
+	return authorHandler{
+		authorsService: as,
+		transformer:    &berthaTransformer{},
+	}
+}
+
+func (ah *authorHandler) getAuthorsCount(writer http.ResponseWriter, req *http.Request) {
+	c, err := ah.authorsService.getAuthorsCount()
+	if err != nil {
+		writeJSONError(writer, err.Error(), http.StatusInternalServerError)
+	}
+	writeJSONResponse(c, true, writer)
+}
+
+func (ah *authorHandler) getAuthorsUuids(writer http.ResponseWriter, req *http.Request) {
+	uuids := ah.authorsService.getAuthorsUuids()
```

Comment:
Actually, it returns an empty author. However, I added a test for this scenario.